modules/shared/partials/community-supported-connectors.adoc

```diff
@@ -1,4 +1,5 @@
-IMPORTANT: *The Redpanda Connectors Docker image is a community-supported artifact*. For support, reach out to the Redpanda team in the community https://redpanda.com/slack[Slack^].
+IMPORTANT: *The Redpanda Connectors Docker image is a community-supported artifact*. For support, reach out to the Redpanda team in the community https://redpanda.com/slack[Slack^]. +
+Are you looking for xref:redpanda-connect:ROOT:about.adoc[Redpanda Connect documentation]?
```

Comment:
@vladoschreiner - I think these connectors are being retired/deprecated soon. Could we update this message so that it's clearer that users should be thinking about using Redpanda Connect instead?

Reply:
Please:
- Add banners, suggesting using Redpanda Connect (ex-Benthos) instead of Kafka Connect
- Add a docs page that explains a/ existence of both b/ Redpanda Connect being the long term solution